script/hassfest/mypy_config.py

```diff
@@ -198,39 +229,47 @@ def generate_and_validate(config: Config) -> str:
     # Disable strict checks for tests
     tests_section = "mypy-tests.*"
     mypy_config.add_section(tests_section)
-    for key in STRICT_SETTINGS:
+    for key in sorted(STRICT_SETTINGS):
         mypy_config.set(tests_section, key, "false")
 
-    for ignored_module in IGNORED_MODULES:
+    for ignored_module in sorted(IGNORED_MODULES):
```

Comment:
```suggestion
    for ignored_module in IGNORED_MODULES:
```

Reply:
Same here. A human might inadvertently un-sort this list, so keep it sorted here.

Reply:
Same here, IGNORED_MODULES is currently empty, and hopefully it will stay that way.